jhove-modules/src/main/java/edu/harvard/hul/ois/jhove/module/wave/FormatChunk.java

```diff
@@ -108,113 +135,157 @@ public boolean readChunk(RepInfo info) throws IOException {
-            aes.setBitrateReduction (compName, "", "", "",
-                qual, Long.toString (bytesPerSecond), "FIXED");
+            aes.setBitrateReduction(compName, "", "", "",
+                qual, Long.toString(bytesPerSecond), "FIXED");
         }
 
-        module.addWaveProperty (new Property ("AverageBytesPerSecond",
+        module.addWaveProperty(new Property("AverageBytesPerSecond",
                     PropertyType.LONG,
-                    new Long (bytesPerSecond)));
-        module.addWaveProperty (new Property ("BlockAlign",
+                    bytesPerSecond));
+        module.addWaveProperty(new Property("BlockAlign",
                     PropertyType.INTEGER,
-                    new Integer (blockAlign)));
+                    blockAlign));
         if (extraBytes != null) {
-            module.addWaveProperty (new Property ("ExtraFormatBytes",
+            module.addWaveProperty(new Property("ExtraFormatBytes",
                     PropertyType.BYTE,
                     PropertyArity.ARRAY,
                     extraBytes));
         }
         if (validBitsPerSample != -1) {
             // Should this property be called something like
             // ValidBitsPersampleOrSamplesPerBlock?
-            module.addWaveProperty (new Property ("ValidBitsPerSample",
+            module.addWaveProperty(new Property("ValidBitsPerSample",
                     PropertyType.INTEGER,
-                    new Integer (validBitsPerSample)));
+                    validBitsPerSample));
         }
         if (channelMask != -1) {
-            module.addWaveProperty (new Property ("ChannelMask",
+            module.addWaveProperty(new Property("ChannelMask",
                     PropertyType.LONG,
-                    new Long (channelMask)));
+                    channelMask));
         }
         if (subformat != null) {
-            module.addWaveProperty (new Property ("Subformat",
-                    PropertyType.BYTE,
-                    PropertyArity.ARRAY,
+            module.addWaveProperty(new Property("Subformat",
+                    PropertyType.STRING,
                     subformat));
         }
+
         return true;
     }
 
-    /* Set default channel assignments.  This is fairly simple,
+    /**
+     * Set default channel assignments. This is fairly simple,
      * but it's helpful to keep the same structure as the equivalent
-     * CommonChunk.setChannelLocations function. */
-    private void setChannelLocations 
-        (AESAudioMetadata aes, int numChannels)
-    {
+     * CommonChunk.setChannelLocations function.
+     */
+    private void setChannelLocations(AESAudioMetadata aes, int numChannels) {
+
         String[] mapLoc = new String[numChannels];
         switch (numChannels) {
             case 2:
-            mapLoc[0] = "LEFT";
-            mapLoc[1] = "RIGHT";
-            break;
-            
+                mapLoc[0] = "LEFT";
+                mapLoc[1] = "RIGHT";
+                break;
+
```

Comment:
Mixed spacing.

Reply:
Looks the same to me... Do you not indent switch cases?